ic10emu/src/vm/instructions/enums.rs

```diff
@@ -0,0 +1,1567 @@
+use serde_derive::{Deserialize, Serialize};
```

Comment:
missing a big disclaimer that this is code-gen, I still feel like it should be put in a path like `src/vm/instructions/codegen/` so that it's clear by just lookit at the file that it's code-gen, instead of only having to scroll up and seeing a disclaimer.

Same for traits.rs

Reply:
apparently `quote` does not preserve comments